redash/handlers/queries.py

```diff
@@ -181,7 +181,9 @@ def require_access_to_dropdown_queries(user, query_def):
         if len(groups) < len(dropdown_query_ids):
             abort(400, message='You are trying to associate a dropdown query that does not have a matching group. Please verify the dropdown query id you are trying to associate with this query.')
 
-        require_access(dict(groups), user, view_only)
+        object = lambda: None
+        object.groups = dict(groups)
+        require_access(object, user, view_only)
```

Comment:
This feels wrong. Let's add `require_access_to_groups` that takes groups?

Btw, `object` is a reserved word.

Reply:
> Btw, object is a reserved word.
You sure? I would expect things to not work if it were a reserved keyword.

Reply:
```
>>> object
<type 'object'>
```

Python is funny in this way. `id` is reserved too and often we override it locally.

Reply:
I guess that reserved word is not the correct term here. Just a built-in object.

Reply:
Yeah, while not reserved, it might cause confusion. I'll rename.